samples/config_helper_vulkan.cc

```diff
@@ -18,6 +18,7 @@
 #include <cassert>
 #include <cstring>
 #include <set>
+#include <vulkan/vulkan.h>
```

Comment:
Hmmm. I normally use " quotes for things not shipped with the compiler.   But others have argued against me.
Coder's choice here.

Reply:
I usually go with ""'s for things in the repo (so in include/ src/ or third_party/) and <> otherwise